lib/directories.py

```diff
@@ -61,9 +61,11 @@ def get_images_data_dir() -> str:
     global _images_data_dir
 
     if _images_data_dir is None:
-        _images_data_dir = get_git_config('--type=path', 'cockpit.bots.images-data-dir')
-
+        _images_data_dir = os.getenv('COCKPIT_IMAGES_DATA_DIR')
         if _images_data_dir is None:
-            _images_data_dir = xdg_cache_home('cockpit-images')
+            _images_data_dir = get_git_config('--type=path', 'cockpit.bots.images-data-dir')
```

Comment:
Does anyone still use this?

Reply:
I don't know -- we've documented it for many years, and I had even had it myself for a long time (when I didn't have enough space in my $HOME). For human users this may feel better than setting an env var.

Reply:
OK.